catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -417,4 +435,18 @@
         </service-properties>
     </service>
 
+    <bean id="resourceDownloadBean" class="ddf.catalog.resource.download.ResourceDownloadEndpoint">
```

Comment:
I was trying to think of a way the resourceDownloadBean could be extracted to its own blueprint and bundle. The goal would have been so ti could be included in a potential product cache feature. However, it seems like the dependency on the reliableResourceManager doesn't really make that possible.